subprojects/docs/src/docs/design/gradle-module-metadata-1.0-specification.md

```diff
@@ -85,6 +85,15 @@ This value must contain an array of 0 or more capabilities. Each capability is a
 #### Standard attributes
 
 - `org.gradle.usage` indicates the purpose of the variant. See the `org.gradle.api.attributes.Usage` class for more details. Value must be a string.
+- `org.gradle.status` indicates the kind of release: one of `release` or `integration`.
+
+#### Java Ecosystem specific attributes
+
+- `org.gradle.dependency.bundling` indicates how dependencies of the variant are bundled. Either externally, embedded or shadowed. See the `org.gradle.api.attributes.java.Bundling` for more details. Value must be a string.
+- `org.gradle.component.category` indicates the type of component (library or platform). This attribute is mostly used to disambiguate Maven POM files derived either as a platform or a library. Value must be a string.
```

Comment:
Is this Java specific? It's used for POM files at the moment, but it's not really specific to that.

Reply:
Moved to the generic attributes